templates/pi-hole-docker-compose.yml.j2

```diff
@@ -27,8 +27,7 @@ services:
       - "443:443/tcp"
     environment:
       TZ: '{{ pihole_timezone }}'
-      WEBPASSWORD: '{{ pihole_password }}'
-      ServerIP: '{{ ansible_facts['default_ipv4']['address'] }}'
```

Comment:
I believe this should be `FTLCONF_dns_reply_host_IPv4` now.

See: https://discourse.pi-hole.net/t/ftlconf-local-ipv4-variable-no-longer-being-used-in-v6/76657/3

Reply:
Good catch. Updated.